bundle/edu.gemini.shared.skyobject/src/main/java/edu/gemini/shared/skyobject/Magnitude.java

```diff
@@ -235,7 +228,7 @@ public int hashCode() {
      */
     @Override
     public int compareTo(Magnitude that) {
-        int res = system.compareTo(that.system);
+        int res = MagnitudeSystem.all().indexOf(that.system) - MagnitudeSystem.all().indexOf(this.system);
```

Comment:
I think the sorting of magnitudes in general is problematic, we need it for  display I guess but is there really a way to in general say this magnitude is "before" the other one if they have different systems?